src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Common/PackageCollection.cs

```diff
@@ -50,11 +50,11 @@ IEnumerator IEnumerable.GetEnumerator()
             Assumes.NotNull(projects);
 
             // Read package references from all projects.
-            IEnumerable<Task<IReadOnlyCollection<IPackageReferenceContextInfo>>> tasks = projects
-                .Select(project => project.GetInstalledPackagesAsync(serviceBroker, cancellationToken).AsTask());
-            IEnumerable<IPackageReferenceContextInfo>[] packageReferences = await Task.WhenAll(tasks);
 
-            return FromPackageReferences(packageReferences.SelectMany(e => e));
+            IReadOnlyDictionary<string, IReadOnlyCollection<IPackageReferenceContextInfo>>? packageReferences =
+                await projects.GetInstalledPackagesAsync(serviceBroker, cancellationToken).ConfigureAwait(true);
```

Comment:
Use named parameter on that pesky boolean literal.

Did you verify that `ConfigureAwait(...)` is beneficial here?

Reply:
I think `ConfigureAwait` is well known enough to not need to name that parameter.

Reply:
Having said that [we should never use `ConfigureAwait(false)` in VS components](https://github.com/microsoft/vs-threading/blob/main/doc/cookbook_vs.md#should-i-await-a-task-with-configureawaitfalse)

Reply:
I'm blaming this on a sneaky VS Quick Refactoring, as the default action is `add ConfigureAwait(true)`, then I ignored it thinking was present in existing code.

Sorry for the distraction!